clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1474,17 +1475,20 @@ public String toHtml(Map<String, String> dynamicUpdateModes) {
                 String updateMode = dynamicUpdateModes.get(key.name);
                 if (updateMode == null)
                     updateMode = "read-only";
-                addConfigDetail(b, "Update Mode", updateMode);
+                addConfigDetail(b, "Update Mode:", updateMode);
             }
-            b.append("</ul>");
-            b.append("</li>\n");
+            b.append("</tbody></table>");
+            b.append("</li>");
         }
         b.append("</ul>\n");
         return b.toString();
     }
 
     private static void addConfigDetail(StringBuilder builder, String name, String value) {
-        builder.append("<li><b>" + name + "</b>: " + value + "</li>");
+        builder.append("<tr style=\"background-color: transparent; padding: 0 0 0 0;\">" +
+                "<th style=\"border-width: 0; background-color: transparent; text-align: right; padding: 0 0 0 0; \">" + name + ":</th>" +
+                "<td style=\"border-width: 0; background-color: transparent; text-align: left; padding: 0 0 0 10;\">" + value + "</td>" +
+                "</tr>");
```

Comment:
Can we add line return after HTML block like tables, rows, list items so the generated HTML stays readable?

Reply:
I've added some newlines to make it more readable. It's still not great due to missing indentation, but I felt that would be taking it too far.